.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,15 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
Can you add the `.idea` folder to the `.gitignore` or remove it from the repo. It is not clear what it is doing here.

Reply:
isn't .gitignore a versioned file?  I looked up some ways to unversion .idea without adding it to .gitignore, so as not to pollute .gitignore.  But if you're ok with it, I can add .idea there.

btw, .idea files are generated by Pycharm.  Surely, I can't be the only one using Pycharm..?

Reply:
I have no objection to adding `.idea` files / folders to the `.gitignore`

Reply:
removed and added .idea to gitignore.